VAPs/squire/intro-to-squire.ipynb

```diff
@@ -0,0 +1,3810 @@
+{
```

Comment:
<p>Qualatative to Qualitative</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-Development/ARM-Notebooks/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@zssherman I addressed this in the recent commit